### PR TITLE
144 tour detail hero block

### LINKED ISSUE #144: Tour - hero block for tour detail pages
In the tour detail pages there is a hero block which shows:
- A breadcrumb
- Title and description of the tour
- Price
- Buy button
- Ticket label (if present)
- The background element/s of the hero, which can be images, videos or combination of both. If there is more than one background element, the controls to move to the next/prev element need to be displayed

Example: https://www.realmadrid.com/tour-bernabeu/premium

Screenshot 2023-06-05 at 13 46 35
Screenshot 2023-06-05 at 13 46 05

### PR DESCRIPTION
**Pull request by msagolj**
Adds the Hero Block at the top of a tour detail page.

Fix #144 

Test URLs:
Normal Tour:
- Before:  https://main--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/premium
- After: https://144-tour-detail-hero-block--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/premium

Combo Tour:
- Before: https://main--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/colegios/combinado-parque-warner
- After: https://144-tour-detail-hero-block--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/colegios/combinado-parque-warner

### CODE REVIEW

**Comment by vdua** [comment]
@msagolj we may need to handle stlying for `rtl` languages. and just fyi we have decided not to use `inset-inline` properties due to lesser browser support but other props like margin-inline, padding-inline should be used.
cc @chicharr

**Comment by chicharr** [comment]
Merging as we need it for content migration. Opened a follow-up ticket to address pending comments:
https://github.com/hlxsites/realmadrid/issues/156

File: sites/blocks/hero-tour-detail/hero-tour-detail.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  // as it differs from title set on the parent page itself
+  resp = await fetch(`${TOUR_LANGUAGE_HOME_PATH[getLanguage()]}/fragments/tour-navigation.plain.html`);
+  if (!resp.ok) return;
+  let parentURL = document.location.pathname;
+  parentURL = parentURL.substring(0, parentURL.lastIndexOf('/'));
+  const groupName = new DOMParser().parseFromString(await resp.text(), 'text/html')
+    .querySelector(`.navigation a[href='${parentURL}']`)?.innerText;
+
+  // get placeholders (non-existing values are undefined)
+  const placeholders = await fetchPlaceholders();
```
[the remark this answers is not in the document]
Comment (msagolj) [reply sites/blocks/hero-tour-detail/hero-tour-detail.js]: done

File: sites/blocks/hero-tour-detail/hero-tour-detail.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        <span class='subtitle'>${Subtitle}</span>
+      </h1>
+    `);
+    dom.querySelector('.product-name').append(title);
+  } else {
+    // with combo image
+    const title = document.createRange().createContextualFragment(`
+      <h1 class='combo-container'>
+        <div class='combo-image-container'>
+          <img src='${comboImage}'>
```
[the remark this answers is not in the document]
Comment (msagolj) [reply sites/blocks/hero-tour-detail/hero-tour-detail.js:281]: added alt text in all occurences of combo image, will be part of PR for #156:
https://github.com/hlxsites/realmadrid/compare/main...156-tour-hero-detail#diff-d18c2fbc7429cdda32bae93ab143797a57becc59dff044b5847762190f0698e2L282-R282

https://github.com/hlxsites/realmadrid/compare/main...156-tour-hero-detail#diff-d18c2fbc7429cdda32bae93ab143797a57becc59dff044b5847762190f0698e2L282-R282

https://github.com/hlxsites/realmadrid/compare/main...156-tour-hero-detail#diff-d18c2fbc7429cdda32bae93ab143797a57becc59dff044b5847762190f0698e2L282-R282

**Pull request merged**